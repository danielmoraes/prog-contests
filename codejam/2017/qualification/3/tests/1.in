7
5 1
5 2
5 4
5 5
6 1
6 3
6 6
