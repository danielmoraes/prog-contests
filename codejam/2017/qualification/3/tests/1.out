Case #1: 2 2
Case #2: 1 0
Case #3: 0 0
Case #4: 0 0
Case #5: 3 2
Case #6: 1 0
Case #7: 0 0
